### DISCUSSION TITLE
Inquiry Regarding Functionality Confirmation After Soldering

### DISCUSSION THREAD

**Discussion by seii-m** [hrvach/deskhop#34] [Q&A]
Thank you for the fantastic project!
I apologize if my explanation is unclear as English is not my strong suit.

The assembly is complete, but it is not functioning properly.
I conducted a connectivity check with a tester, and it appears that the soldering itself has been done successfully.
If you have any suggestions for effective testing methods or any other advice, please let me know.

**Comment by hrvach** [comment]
Try this:

1. Check if proper uf2 is written to the proper board (A to A, B to B)
2. Plug keyboard in A, plug both picos to computers, keep pressing caps lock - leds should alternate.
3. Plug keyboard in B, repeat 2. - leds should alternate

What happens in your case?

**Reply by seii-m** [reply]
OK => 1. Check if proper uf2 is written to the proper board (A to A, B to B)
NG => 2. Plug keyboard in A, plug both picos to computers, keep pressing caps lock - leds should alternate.
NG => 3. Plug keyboard in B, repeat 2. - leds should alternate

Hmm...
It's possible that the ADuM1201BRZ was a defective product...
I will procure the ADuM1201BRZ again.

**Comment by hrvach** [comment]
@seii-m does any single LED light up when you press caps lock in this scenario? Both picos need to be connected to their respective computers (can be the same machine).

Check https://github.com/hrvach/deskhop/wiki/Troubleshooting for more suggestions, maybe it helps.

**Comment by seii-m** [comment]
When I placed the reorder...

Finally, I did it!
Thank you for the fantastic project!
